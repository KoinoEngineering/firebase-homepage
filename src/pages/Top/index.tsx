import React from "react";
import PageContainer from "src/components/templates/Page/PageContainer";
import Profile from "src/components/templates/Profile";

const Top: React.FC = () => {
    return <PageContainer id="Top">
        <Profile />
    </PageContainer>;
};

export default Top;