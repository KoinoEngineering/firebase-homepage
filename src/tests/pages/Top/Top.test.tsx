import { render } from "@testing-library/react";
import React from "react";
import Top from "src/pages/Top";

jest.mock("src/components/templates/Profile", () => {
    const Profile = () => <div>Profile</div>;
    return Profile;
});

test("Top expect to match snapshot", () => {
    expect(render(<Top />)).toMatchSnapshot();
});
