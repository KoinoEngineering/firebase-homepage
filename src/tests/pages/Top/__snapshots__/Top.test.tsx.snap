// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Top expect to match snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="MuiContainer-root TopPageContainer-root-2 MuiContainer-maxWidthLg"
        id="Top"
      >
        <div
          class="MuiGrid-root MuiGrid-container"
        >
          <div
            class="MuiGrid-root MuiGrid-container"
          >
            <div
              class="MuiGrid-root MuiGrid-container"
              id="Profile"
            >
              <div
                class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-justify-xs-center MuiGrid-grid-xs-12 MuiGrid-grid-sm-true"
              >
                <img
                  alt="こいのエンジニアリング"
                  src="logo.svg"
                  width="100%"
                />
              </div>
              <div
                class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-true"
              >
                <ul
                  class="MuiList-root MuiList-padding MuiList-subheader"
                >
                  <li
                    class="MuiListSubheader-root MuiListSubheader-gutters"
                  >
                    <h2
                      class="MuiTypography-root MuiTypography-h2"
                    >
                      プロフィール
                    </h2>
                  </li>
                  <div
                    aria-disabled="false"
                    class="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
                    role="button"
                    tabindex="0"
                  >
                    名前: こいのエンジニアリング
                  </div>
                  <div
                    aria-disabled="false"
                    class="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
                    role="button"
                    tabindex="0"
                  >
                    開発年数: 2014年8月～現在(5年7ヶ月15日)
                  </div>
                  <div
                    aria-disabled="false"
                    class="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
                    role="button"
                    tabindex="0"
                  >
                    <ul
                      class="MuiList-root MuiList-padding MuiList-subheader"
                    >
                      <li
                        class="MuiListSubheader-root MuiListSubheader-gutters"
                      >
                        <h3
                          class="MuiTypography-root MuiTypography-h3"
                        >
                          できること
                        </h3>
                      </li>
                      <div
                        aria-disabled="false"
                        class="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
                        role="button"
                        tabindex="0"
                      >
                        <ul
                          class="MuiList-root MuiList-padding MuiList-subheader"
                        >
                          <li
                            class="MuiListSubheader-root MuiListSubheader-gutters"
                          >
                            <h3
                              class="MuiTypography-root MuiTypography-h3"
                            >
                              言語
                            </h3>
                          </li>
                          <div
                            aria-disabled="false"
                            class="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
                            role="button"
                            tabindex="0"
                          >
                            typescript
                          </div>
                          <div
                            aria-disabled="false"
                            class="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
                            role="button"
                            tabindex="0"
                          >
                            react/redux/redux-saga
                          </div>
                          <div
                            aria-disabled="false"
                            class="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
                            role="button"
                            tabindex="0"
                          >
                            java
                          </div>
                          <div
                            aria-disabled="false"
                            class="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
                            role="button"
                            tabindex="0"
                          >
                            C#
                          </div>
                        </ul>
                      </div>
                      <div
                        aria-disabled="false"
                        class="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
                        role="button"
                        tabindex="0"
                      >
                        <ul
                          class="MuiList-root MuiList-padding MuiList-subheader"
                        >
                          <li
                            class="MuiListSubheader-root MuiListSubheader-gutters"
                          >
                            <h3
                              class="MuiTypography-root MuiTypography-h3"
                            >
                              環境
                            </h3>
                          </li>
                          <div
                            aria-disabled="false"
                            class="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
                            role="button"
                            tabindex="0"
                          >
                            firebase
                          </div>
                          <div
                            aria-disabled="false"
                            class="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
                            role="button"
                            tabindex="0"
                          >
                            heroku
                          </div>
                          <div
                            aria-disabled="false"
                            class="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
                            role="button"
                            tabindex="0"
                          >
                            aws(勉強中)
                          </div>
                        </ul>
                      </div>
                    </ul>
                  </div>
                </ul>
              </div>
            </div>
          </div>
          <div
            class="MuiGrid-root MuiGrid-container"
          >
            <div
              class="MuiGrid-root MuiGrid-container"
            >
              <h2
                class="MuiTypography-root MuiTypography-h2"
              >
                作ったものとか
              </h2>
            </div>
            <div
              class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-4"
            >
              <div
                class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4"
              >
                <div
                  class="MuiPaper-root MuiCard-root MuiPaper-elevation8 MuiPaper-rounded"
                >
                  <a
                    aria-disabled="false"
                    class="MuiButtonBase-root MuiCardActionArea-root"
                    href="https://www.koino.engineering/"
                    rel="noopener noreferrer"
                    tabindex="0"
                    target="_blank"
                  >
                    <div
                      class="MuiCardHeader-root"
                    >
                      <div
                        class="MuiCardHeader-content"
                      >
                        <span
                          class="MuiTypography-root MuiCardHeader-title MuiTypography-h5 MuiTypography-displayBlock"
                        >
                          旧ホームページ(廃止予定)
                        </span>
                      </div>
                    </div>
                    <span
                      class="MuiCardActionArea-focusHighlight"
                    />
                    <span
                      class="MuiTouchRipple-root"
                    />
                  </a>
                </div>
              </div>
              <div
                class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4"
              >
                <div
                  class="MuiPaper-root MuiCard-root MuiPaper-elevation8 MuiPaper-rounded"
                >
                  <button
                    class="MuiButtonBase-root MuiCardActionArea-root"
                    tabindex="0"
                    type="button"
                  >
                    <div
                      class="MuiCardHeader-root"
                    >
                      <div
                        class="MuiCardHeader-content"
                      >
                        <span
                          class="MuiTypography-root MuiCardHeader-title MuiTypography-h5 MuiTypography-displayBlock"
                        >
                          Webサイト(ここ)
                        </span>
                      </div>
                    </div>
                    <span
                      class="MuiCardActionArea-focusHighlight"
                    />
                    <span
                      class="MuiTouchRipple-root"
                    />
                  </button>
                </div>
              </div>
              <div
                class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4"
              >
                <div
                  class="MuiPaper-root MuiCard-root MuiPaper-elevation8 MuiPaper-rounded"
                >
                  <a
                    aria-disabled="false"
                    class="MuiButtonBase-root MuiCardActionArea-root"
                    href="https://react-study.netlify.com/"
                    rel="noopener noreferrer"
                    tabindex="0"
                    target="_blank"
                  >
                    <div
                      class="MuiCardHeader-root"
                    >
                      <div
                        class="MuiCardHeader-content"
                      >
                        <span
                          class="MuiTypography-root MuiCardHeader-title MuiTypography-h5 MuiTypography-displayBlock"
                        >
                          リアクト勉強部屋
                        </span>
                      </div>
                    </div>
                    <span
                      class="MuiCardActionArea-focusHighlight"
                    />
                    <span
                      class="MuiTouchRipple-root"
                    />
                  </a>
                </div>
              </div>
              <div
                class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4"
              >
                <div
                  class="MuiPaper-root MuiCard-root MuiPaper-elevation8 MuiPaper-rounded"
                >
                  <a
                    aria-disabled="false"
                    class="MuiButtonBase-root MuiCardActionArea-root"
                    href="https://mazegenerator.netlify.com/"
                    rel="noopener noreferrer"
                    tabindex="0"
                    target="_blank"
                  >
                    <div
                      class="MuiCardHeader-root"
                    >
                      <div
                        class="MuiCardHeader-content"
                      >
                        <span
                          class="MuiTypography-root MuiCardHeader-title MuiTypography-h5 MuiTypography-displayBlock"
                        >
                          迷路
                        </span>
                      </div>
                    </div>
                    <span
                      class="MuiCardActionArea-focusHighlight"
                    />
                    <span
                      class="MuiTouchRipple-root"
                    />
                  </a>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="MuiContainer-root TopPageContainer-root-2 MuiContainer-maxWidthLg"
      id="Top"
    >
      <div
        class="MuiGrid-root MuiGrid-container"
      >
        <div
          class="MuiGrid-root MuiGrid-container"
        >
          <div
            class="MuiGrid-root MuiGrid-container"
            id="Profile"
          >
            <div
              class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-justify-xs-center MuiGrid-grid-xs-12 MuiGrid-grid-sm-true"
            >
              <img
                alt="こいのエンジニアリング"
                src="logo.svg"
                width="100%"
              />
            </div>
            <div
              class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-true"
            >
              <ul
                class="MuiList-root MuiList-padding MuiList-subheader"
              >
                <li
                  class="MuiListSubheader-root MuiListSubheader-gutters"
                >
                  <h2
                    class="MuiTypography-root MuiTypography-h2"
                  >
                    プロフィール
                  </h2>
                </li>
                <div
                  aria-disabled="false"
                  class="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
                  role="button"
                  tabindex="0"
                >
                  名前: こいのエンジニアリング
                </div>
                <div
                  aria-disabled="false"
                  class="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
                  role="button"
                  tabindex="0"
                >
                  開発年数: 2014年8月～現在(5年7ヶ月15日)
                </div>
                <div
                  aria-disabled="false"
                  class="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
                  role="button"
                  tabindex="0"
                >
                  <ul
                    class="MuiList-root MuiList-padding MuiList-subheader"
                  >
                    <li
                      class="MuiListSubheader-root MuiListSubheader-gutters"
                    >
                      <h3
                        class="MuiTypography-root MuiTypography-h3"
                      >
                        できること
                      </h3>
                    </li>
                    <div
                      aria-disabled="false"
                      class="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
                      role="button"
                      tabindex="0"
                    >
                      <ul
                        class="MuiList-root MuiList-padding MuiList-subheader"
                      >
                        <li
                          class="MuiListSubheader-root MuiListSubheader-gutters"
                        >
                          <h3
                            class="MuiTypography-root MuiTypography-h3"
                          >
                            言語
                          </h3>
                        </li>
                        <div
                          aria-disabled="false"
                          class="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
                          role="button"
                          tabindex="0"
                        >
                          typescript
                        </div>
                        <div
                          aria-disabled="false"
                          class="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
                          role="button"
                          tabindex="0"
                        >
                          react/redux/redux-saga
                        </div>
                        <div
                          aria-disabled="false"
                          class="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
                          role="button"
                          tabindex="0"
                        >
                          java
                        </div>
                        <div
                          aria-disabled="false"
                          class="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
                          role="button"
                          tabindex="0"
                        >
                          C#
                        </div>
                      </ul>
                    </div>
                    <div
                      aria-disabled="false"
                      class="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
                      role="button"
                      tabindex="0"
                    >
                      <ul
                        class="MuiList-root MuiList-padding MuiList-subheader"
                      >
                        <li
                          class="MuiListSubheader-root MuiListSubheader-gutters"
                        >
                          <h3
                            class="MuiTypography-root MuiTypography-h3"
                          >
                            環境
                          </h3>
                        </li>
                        <div
                          aria-disabled="false"
                          class="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
                          role="button"
                          tabindex="0"
                        >
                          firebase
                        </div>
                        <div
                          aria-disabled="false"
                          class="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
                          role="button"
                          tabindex="0"
                        >
                          heroku
                        </div>
                        <div
                          aria-disabled="false"
                          class="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
                          role="button"
                          tabindex="0"
                        >
                          aws(勉強中)
                        </div>
                      </ul>
                    </div>
                  </ul>
                </div>
              </ul>
            </div>
          </div>
        </div>
        <div
          class="MuiGrid-root MuiGrid-container"
        >
          <div
            class="MuiGrid-root MuiGrid-container"
          >
            <h2
              class="MuiTypography-root MuiTypography-h2"
            >
              作ったものとか
            </h2>
          </div>
          <div
            class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-4"
          >
            <div
              class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4"
            >
              <div
                class="MuiPaper-root MuiCard-root MuiPaper-elevation8 MuiPaper-rounded"
              >
                <a
                  aria-disabled="false"
                  class="MuiButtonBase-root MuiCardActionArea-root"
                  href="https://www.koino.engineering/"
                  rel="noopener noreferrer"
                  tabindex="0"
                  target="_blank"
                >
                  <div
                    class="MuiCardHeader-root"
                  >
                    <div
                      class="MuiCardHeader-content"
                    >
                      <span
                        class="MuiTypography-root MuiCardHeader-title MuiTypography-h5 MuiTypography-displayBlock"
                      >
                        旧ホームページ(廃止予定)
                      </span>
                    </div>
                  </div>
                  <span
                    class="MuiCardActionArea-focusHighlight"
                  />
                  <span
                    class="MuiTouchRipple-root"
                  />
                </a>
              </div>
            </div>
            <div
              class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4"
            >
              <div
                class="MuiPaper-root MuiCard-root MuiPaper-elevation8 MuiPaper-rounded"
              >
                <button
                  class="MuiButtonBase-root MuiCardActionArea-root"
                  tabindex="0"
                  type="button"
                >
                  <div
                    class="MuiCardHeader-root"
                  >
                    <div
                      class="MuiCardHeader-content"
                    >
                      <span
                        class="MuiTypography-root MuiCardHeader-title MuiTypography-h5 MuiTypography-displayBlock"
                      >
                        Webサイト(ここ)
                      </span>
                    </div>
                  </div>
                  <span
                    class="MuiCardActionArea-focusHighlight"
                  />
                  <span
                    class="MuiTouchRipple-root"
                  />
                </button>
              </div>
            </div>
            <div
              class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4"
            >
              <div
                class="MuiPaper-root MuiCard-root MuiPaper-elevation8 MuiPaper-rounded"
              >
                <a
                  aria-disabled="false"
                  class="MuiButtonBase-root MuiCardActionArea-root"
                  href="https://react-study.netlify.com/"
                  rel="noopener noreferrer"
                  tabindex="0"
                  target="_blank"
                >
                  <div
                    class="MuiCardHeader-root"
                  >
                    <div
                      class="MuiCardHeader-content"
                    >
                      <span
                        class="MuiTypography-root MuiCardHeader-title MuiTypography-h5 MuiTypography-displayBlock"
                      >
                        リアクト勉強部屋
                      </span>
                    </div>
                  </div>
                  <span
                    class="MuiCardActionArea-focusHighlight"
                  />
                  <span
                    class="MuiTouchRipple-root"
                  />
                </a>
              </div>
            </div>
            <div
              class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4"
            >
              <div
                class="MuiPaper-root MuiCard-root MuiPaper-elevation8 MuiPaper-rounded"
              >
                <a
                  aria-disabled="false"
                  class="MuiButtonBase-root MuiCardActionArea-root"
                  href="https://mazegenerator.netlify.com/"
                  rel="noopener noreferrer"
                  tabindex="0"
                  target="_blank"
                >
                  <div
                    class="MuiCardHeader-root"
                  >
                    <div
                      class="MuiCardHeader-content"
                    >
                      <span
                        class="MuiTypography-root MuiCardHeader-title MuiTypography-h5 MuiTypography-displayBlock"
                      >
                        迷路
                      </span>
                    </div>
                  </div>
                  <span
                    class="MuiCardActionArea-focusHighlight"
                  />
                  <span
                    class="MuiTouchRipple-root"
                  />
                </a>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
